vxquery-xtest/src/main/java/org/apache/vxquery/xtest/TestRunner.java

```diff
@@ -167,6 +180,16 @@ public TestCaseResult run(final TestCase testCase) {
             long end = System.currentTimeMillis();
             res.time = end - start;
         }
+        if (opts.showResult) {
+            if (res.result == null) {
+                System.err.println("***Error: ");
+                System.err.println(res.error.getStackTrace());
+            } else {
+                System.err.println("***Result: ");
+                System.err.println(res.result);
+
```

Comment:
Watch your white space. Make sure your adding extra lines only where they improve the ability to read the code. For example, when separating coding blocks.